main.py

```
@@ -1,23 +1,49 @@
+    try:
+        tab = t.transcribe_from_youtube(url)
+
+        # 結果をコンソールに表示
+        print("\n=== TAB ===")
+        print(tab.to_text())
+
+        # 画像として保存
+        tab.to_matplotlib(args.output)
+        print(f"Saved visualization to {args.output}")
+
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)